nbproject/project.properties

```diff
@@ -0,0 +1,7 @@
+include.path=${php.global.include.path}
```

Comment:
Could you remove any of these IDE-specific files from the PR (I imagine they probably need to be gitignored...)